test/04-translations.test.ts

```diff
@@ -0,0 +1,340 @@
+const configPath = __dirname + '/fixtures/04-translations/config';
+const hooksPath = __dirname + '/fixtures/04-translations/translations/hooks';
+import { testInit, testDeInit, testLocalServer } from './lib/test-init';
+import { faker } from '@faker-js/faker';
+import { expect } from 'chai';
+import { AnyObject } from 'pinejs-client-core';
+
+import { PineTest } from 'pinejs-client-supertest';
+
+describe('04 native translation tests', function () {
+	let pineServer: Awaited<ReturnType<typeof testInit>>;
+	let pineTest: PineTest;
+	let faculty: AnyObject;
+	before(async () => {
+		pineServer = await testInit({
+			configPath,
+			hooksPath,
+			deleteDb: true,
+		});
+		// setup faker so that test date uniqueness is set for all test cases
+		faker.seed();
+
+		pineTest = new PineTest({}, { app: testLocalServer });
+
+		// initial faculty is needed, as translations should not create faculties just from name property.
+		// expected to fail when faculty with name is not existing
+		({ body: faculty } = await pineTest
+			.post({
+				apiPrefix: 'university/',
+				resource: 'faculty',
+				body: {
+					name: faker.random.words(2),
+				},
+			})
+			.expect(201));
+	});
+
+	after(async () => {
+		await testDeInit(pineServer);
```

Comment:
```suggestion
	after(() => {
		testDeInit(pineServer);
```

Pretty sure we don't need `await`s for `testDeInit()` calls.